packages/pipeline/src/Stage.tsx

```diff
@@ -34,7 +34,7 @@ export interface StageProps {
   /**
    * The DOM node to use as the root node for the intersectionObserver. Defaults to window when null or undefined.
    **/
-  intersectionNode?: Element;
+  intersectionNode: HTMLElement | null;
```

Comment:
Based on the TSDoc annotation, this should be `intersectionNode?: HTMLElement | null;`, right?

Reply:
Also, this probably needs a patch changeset, even though it's small.

Reply:
sgtm!